### PR TITLE
Speed up univ3 initialization

### PR DESCRIPTION
**Pull request by MartinquaXD**
Related to #1525
Reverts #1719

Reducing the batch size actually does not improve the situation.
However, I noticed that for univ3 initialization we first fetch all the known pools, then sort based on total value locked in Rust and then pick the top n.
What we can do instead is to sort in the query and only fetch the top n directly in the queries.
This reduced start up time by ~15s or 3 requests (compared to old logic with batch size 1000).

Misc changes:
- changed the code to use `usize` for the max init size as this the idiomatic type in rust for sizes of collections.
- made sure the new driver uses the same default init size as the old driver
- removed total value locked from our data types

### Test Plan
Manually ran the start up sequence to compare start up times.

### CODE REVIEW

File: crates/driver/src/infra/config/file/mod.rs [hunk 1]
```diff
@@ -306,8 +306,8 @@ pub enum UniswapV3Preset {
 }
 
 mod uniswap_v3 {
-    pub fn default_max_pools_to_initialize() -> u64 {
-        50
+    pub fn default_max_pools_to_initialize() -> usize {
+        100
```
Comment (fleupold) [inline crates/driver/src/infra/config/file/mod.rs:310]: intended?

File: crates/shared/src/sources/balancer_v2/graph_api.rs [hunk 2]
```diff
@@ -61,6 +61,7 @@ impl BalancerSubgraphClient {
         // suggested approach to paging best performance:
         // <https://thegraph.com/docs/graphql-api#pagination>
         loop {
+            tracing::error!("fetch ");
```
Comment (fleupold) [inline crates/shared/src/sources/balancer_v2/graph_api.rs:64]: remove?

**Review by fleupold — APPROVED** [review]
Really nice optimisations. Would it be beneficial to fetch the tick information in the same query as well?

File: crates/driver/src/infra/config/file/mod.rs (hunk 1 above)
Comment (MartinquaXD) [reply crates/driver/src/infra/config/file/mod.rs:310]: Yes, the default we use in the current driver is `100` so I wanted to make that the same so we don't get surprised when we switch over to the new driver.

**Comment by MartinquaXD** [comment]
> Would it be beneficial to fetch the tick information in the same query as well?

The idea makes sense to me but it's hard to say without testing it out. Ideally we want to have fewer faster requests. So when the graph is having issues we might still get by.
I'm worried that fetching the ticks will make that query so slow that it's more prone to timeouts again. But I will try it out.

**Review by sunce86 — APPROVED** [review]
Can you manually run some of the unit tests for fetching the graph data and confirm that the ordering by `totalValueLockedETH` is done correctly by the graph?
I vaguely remember that the ordering was messy because not all pools had logical values for `totalValueLockedETH`. That's why I added the filtering: 

`.filter(|pool: &PoolData| pool.total_value_locked_eth.is_normal())`

**Comment by sunce86** [comment]
I might suspect that the ordering was done badly by graph which returns some small or empty pools for which fetching the data is much quicker than for deep pools. Other than that I have no clue how the 15s reducement is achieved.

**Comment by MartinquaXD** [comment]
> I might suspect that the ordering was done badly by graph which returns some small or empty pools for which fetching the data is much quicker than for deep pools. Other than that I have no clue how the 15s reducement is achieved.

When I ran the code the graph consistently needed ~5s per request fetching pools. Because we now only fetch the top 100 pools instead of all of them we have to do fewer requests.
Ideally nothing should change about the request fetching the ticks because they should still fetch data for the same 100 pools. (I still need to verify that ordering of the graph is actually correct)

**Pull request merged**